download.sh

```diff
@@ -57,7 +57,7 @@ for f in ${FILES}; do
     TARGETS[${#TARGETS[*]}]="${TARGET_PATH}"
     if [ ! -f "${HOME}/${TARGET_PATH}" ]
     then
-        curl -o "${HOME}/${TARGET_PATH}" "${URL}/${TRACK}/${f}"
+        curl -o "${HOME}/${TARGET_PATH}" -L "${URL}/${TRACK}/${f}"
```

Comment:
I still don't understand why we'd need this but why not :shrug:

Reply:
If we want to be even more solid, what are you thoughts about this?

```suggestion
        curl -fsSL o "${HOME}/${TARGET_PATH}" "${URL}/${TRACK}/${f}"
```

fail, silent, show error, redirect

Reply:
Right it's going to be interactive in various situations.